xpath_test.go

```diff
@@ -30,6 +30,14 @@ func TestCompile(t *testing.T) {
 		t.Fatalf("/a/b/(c, .[not(c)]) should be correct but got error %s", err)
 	}
 }
+
+func TestCompile2(t *testing.T) {
+	_, err := Compile("//ul/li/@clášs日本語")
+	if err != nil {
+		t.Fatalf("//ul/li/@clášs日本語 should be correct but got error %s", err)
+	}
+}
```

Comment:
While this does seem to cover the additions you made in scanner/parse.go code, it would be nice to actually have a test/tests to show the compiled utf8 friendly expr that actually works on an actual html/xml, i.e. select works.